packages/react-components/src/components/ChatMessageComponent.tsx

```diff
@@ -86,11 +87,10 @@ const GenerateTextMessageContent = (payload: ChatMessagePayload): JSX.Element =>
 };
 
 export const ChatMessageComponent = (props: ChatMessageProps): JSX.Element => {
-  const strings = useLocale().strings.messageThread;
   const ids = useIdentifiers();
   const theme = useTheme();
 
-  const { message, onUpdateMessage, onDeleteMessage, editDisabled, showDate, messageContainerStyle } = props;
+  const { message, onUpdateMessage, onDeleteMessage, editDisabled, showDate, messageContainerStyle, strings } = props;
```

Comment:
The `strings` field should always be optional and the implementation should fallback to the strings from locale. See `ParticipantItem` for an examples.

Reply:
This is not a component we are going to expose, this component is just for code/logic separation, so it is always asking for strings from messageThread component